samples/emoji-search/ios/swiftUI/EmojiSearchApp/Widget/ColumnBinding.swift

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (C) 2022 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Foundation
+import shared
+import SwiftUI
+
+final class ColumnBinding: BaseWidget, WidgetColumn, SwiftUIViewBinding {
+    func height(height: Int32) {
```

Comment:
Oof yeah Kotlin value classes don't interop well with Swift unfortunately. On the JVM (Android) these let us define 0 overhead enums.

Reply:
I think we might want to start considering using primitives and language features that do have better inter-opt, since the framework is meant to target both platforms. are there alternatives that could be considered for some of these?

Reply:
We could convert these properties to enums, though that has a small performance impact. Initially the plan was these interfaces would only be implemented in Kotlin.